### DISCUSSION TITLE
Double tap to toggle focus

### DISCUSSION THREAD

**Discussion by Axillman** [lnreader/lnreader#369] [Ideas]
Instead of a single tap, require a double tap to toggle the focus in the reader.

**Comment by Svk1190** [comment]
If you are bothered by the vertical seekbar (it pops up by a single touch and prevents you from reading) you can change it to a horizontal seekbar. The horizontal seekbar shows up at the bottom and does not block you from reading. This can be helpful for now.

I think that single tap to show menu is helpful for me personally, and this is the default setting in other apps like Tachiyomi. Maybe there can be an optional setting which allows the user to choose between 'single tap for menu' and 'double tap for menu'.

Another option might be to have a small menu button (three dots on top right, or bottom left/right corner) such that you need to click on that small menu button to show the menu. This can also reduce the chance of accidentally opening the menu and seekbar.
